PendingReleaseNotes

```diff
@@ -7,3 +7,10 @@
   for more details on the background of this change.
 
 * Nautilus-based librbd clients can now open images on Jewel clusters.
+
```

Comment:
I think we'll want to add a section for 14.2.2

Reply:
Updated